tests/cnf/ran/ptp/internal/cluster/cluster.go

```
@@ -0,0 +1,416 @@
+// exactly one client interface with Ts2phc+Phc2sys (a receiver), or zero client interfaces with
+// ControllingProfileName set (a transmitter).
+func isClockCandidate(parsed *parsedResource) bool {
+	clientCount := 0
+
+	for _, interfaceInfo := range parsed.Data.Interfaces {
+		if interfaceInfo.Role == profiles.InterfaceRoleClient {
+			clientCount++
+		}
+	}
+
+	if clientCount == 1 && hasProcess(parsed.Data.Processes, processes.Ts2phc) && hasProcess(parsed.Data.Processes, processes.Phc2sys) {
```

golangci-lint (2.12.2)

[error] The line is 133 characters long, which exceeds the maximum of 120 characters.

(lll)